src/experimental/safe_html_transform/configure.zcml

```diff
@@ -33,4 +33,9 @@
         handler=".setuphandlers.post_install">
     </genericsetup:importStep>
 
+  <include package="plone.app.registry" />
+  <include package="Products.MimetypesRegistry" />
+  <include package="Products.PortalTransforms" />
+  <include package="Products.PortalTransforms" />
```

Comment:
These shouldn't be necessary, as Plone will include the necessary ZCML for you in all of these cases. You definitely don't need to include ``Products.PortalTransforms`` twice.